assets/src/edit-story/components/canvas/layout.js

```diff
@@ -247,6 +247,18 @@ const CarouselArea = styled(Area).attrs({
   showOverflow: true,
 })``;
 
+const RightClickPageZone = styled.div`
+  position: relative;
+  grid-area: p;
+`;
+
+const RightClickMenuContainer = styled.div`
+  position: absolute;
+  top: ${({ position }) => position?.y ?? 0}px;
+  left: ${({ position }) => position?.x ?? 0}px;
+  z-index: 9999;
```

Comment:
puts this above the quick actions menu. That z index is 9991 (in design system).

Reply:
do you think we need to make the zindex a prop for the quick actions menu that can be overridden?

Reply:
For the right click menu or the quick actions menu?

Reply:
oh i just meant in general since you had to bump it up here to 9999. it's probably fine.